infrastructure/terraform.tfvars.tmpl

```diff
@@ -1,22 +1,18 @@
-aws_profile="${AWS_PROFILE}"
 aws_region="${AWS_REGION}"
 prefix="${PREFIX}"
 vpc_id="${VPC_ID}"
 subnet_tagname="${SUBNET_TAGNAME}"
-iam_role_permissions_boundary="${IAM_ROLE_PERMISSIONS_BOUNDARY}"
+iam_policy_permissions_boundary_name="${PERMISSIONS_BOUNDARY_POLICY_NAME:-null}"
 stage="${STAGE:-dev}"
 assume_role_arns=$ASSUME_ROLE_ARNS
 cognito_app_secret="${COGNITO_APP_SECRET}"
```

Comment:
```suggestion
cognito_app_secret="${VEDA_PROGRAMMATIC_CLIENT_SECRET_ID}"
```

Reply:
i agree that `cognito_app_secret` is confusing, but that's just internally used. The env var name is gonna be `VEDA_PROGRAMMATIC_CLIENT_SECRET_ID` - coming from `veda-auth` via `veda-deploy`.

Reply:
> [!CAUTION]
Note to update the repository secrets before merging to dev.